src/gui_common/RadialPopup.tscn

```diff
@@ -1,25 +1,29 @@
-[gd_scene load_steps=3 format=2]
+[gd_scene load_steps=4 format=2]
 
 [ext_resource path="res://src/gui_common/RadialPopup.cs" type="Script" id=1]
 [ext_resource path="res://src/gui_common/RadialMenu.tscn" type="PackedScene" id=2]
+[ext_resource path="res://src/gui_common/dialogs/CustomDialog.tscn" type="PackedScene" id=3]
 
-[node name="RadialPopup" type="Popup"]
+[node name="RadialPopup" instance=ExtResource( 3 )]
 anchor_right = 1.0
 anchor_bottom = 1.0
-popup_exclusive = true
+margin_right = 0.0
+margin_bottom = 0.0
 script = ExtResource( 1 )
+Exclusive = true
+ExclusiveAllowCloseOnEscape = false
```

Comment:
I think it would be nice for the cancel (ESC) button to also dismiss a radial popup so it is easier to get out of one in case it gets stuck on.